src/modules/endpoint/reducers/cityContentReducer.js

```diff
@@ -15,8 +15,8 @@ export default (
       return pushCategory(state, action)
     case 'PUSH_EVENT':
       return pushEvent(state, action)
-    case 'SWITCH_CITY_CONTENT_LANGUAGE': // todo:  Not working
-      return switchLanguage(state, action)
+    case 'MORPH_CONTENT_LANGUAGE':
```

Comment:
morph? 🤨 I like "switch" better. Even "change" sounds better to me...

Reply:
I tried to use something extraordinary. `SWITCH_` is already used. See the changes for that.
Change is too generic in my opinion. Also the term Morph is used in Computing: https://en.wikipedia.org/wiki/Morph#Technology

We have a problem because we need 2 terms for kind of the same thing: Like "PUSH" and "FETCH" there is "MORPH" and "SWITCH"